README.md

```diff
@@ -55,7 +55,8 @@ There are several ways to provide credentials.
 ### Environment variables
 If you do not specify a profile when running the tool, or the profile you specify does not contain credentials, the
 tool will use the following environment variables. This enables tools such as HashiCorp's Vault to work seamlessly
-with the tool.
+with the tool. <br/>
+**These environment variables will not be utilized if you have a default profile configured. You must remove the default profile in order to utilize theses environment variables within this script.**
```

Comment:
nit `the(?)` instead of `theses`

Reply:
@intertwine - yeah i was trying to figure out which to use as well. i went with these because right after this sentence, we list out the environment variables we'll use. do you think the is still better in this context? i don't have a strong opinion either way.

Reply:
How about something like "You must remove the default profile to use the following environment variables within this script:"